doc/releases/changelog-dev.md

```diff
@@ -287,6 +287,10 @@
 
   The i-th element in the list refers to the 5 samples generated by the i-th measurement of the algorithm.
 
+  <h4>Qutit mixed state initialize_state module added 🧩</h4>
```

Comment:
```suggestion
```

Removing this mostly just because the sectioning of the changelog is handled by the product team.